src/style.css

```
@@ -0,0 +1,24 @@
+html {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+  
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+  
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+p {
+  margin: 0;
+}
```

```suggestion
h1,
p {
  margin: 0;
}
```
Don't repeat yourself. You have `margin: 0` both in your `h1` and `p` selectors. You may write it like I've shown above and remove the rule `margin: 0` from your `h1` tag